package.json

```diff
@@ -59,7 +59,7 @@
         "gulp-insert": "latest",
         "gulp-newer": "latest",
         "gulp-sourcemaps": "latest",
-        "gulp-typescript": "3.1.5",
+        "gulp-typescript": "latest",
```

Comment:
invalid peer dependency, now both are conflicting. i have pushed https://github.com/ivogabe/gulp-typescript/pull/510 to fix it.